### ISSUE TITLE
does it work for now?

### ISSUE THREAD

**Issue by wade0317** [gy2006/flow-platform-x#6]
Does it work for now?   Can i make flow-platform-x replace flow.ci ?

**Comment by gy2006** [comment]
Yes, please see the release notes https://github.com/FlowCI/flow-platform-x/releases
